client/components/badge/docs/example.jsx

```diff
@@ -21,6 +21,12 @@ BadgeExample.defaultProps = {
 		<div>
 			<Badge type="success">Success Badge</Badge>
 			<Badge type="warning">Warning Badge</Badge>
+			<Badge type="success" size="small">
```

Comment:
Can we use `compact` here like the Button component? For example:

`<Badge type="success" compact>`